packages/manager/apps/catalog/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@ovh-ux/manager-catalog-app",
+  "version": "1.0.0",
+  "private": true,
+  "description": "catalog revamp react app",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/apps/catalog-revamp"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "scripts": {
+    "build": "tsc && vite build",
+    "dev": "tsc && vite",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-catalog-app' --include-dependencies -- npm run build --if-present",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-catalog-app' --include-dependencies -- npm run dev --if-present",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-catalog-app' --include-dependencies -- npm run dev:watch --if-present",
+    "pretest": "tsc -p tsconfig.e2e.json",
+    "test": "jest --coverage -u",
+    "test:e2e": "tsc && npx playwright test --headed",
+    "test:e2e:ci": "tsc && npx playwright test",
+    "test:e2e:script": "tsc && node ../../../../scripts/run-playwright.js"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-core-api": "*",
+    "@ovh-ux/manager-react-core-application": "*",
+    "@ovh-ux/shell": "^3.0.0",
+    "@ovhcloud/msc-react-tile": "*",
+    "@ovhcloud/msc-utils": "*",
+    "@ovhcloud/ods-common-theming": "16.0.1",
+    "@ovhcloud/ods-components": "16.0.1",
+    "@ovhcloud/ods-theme-blue-jeans": "16.0.1",
+    "@tanstack/react-query": "^4.29.3",
+    "@tanstack/react-query-devtools": "^4.9.0",
+    "autoprefixer": "^10.4.16",
+    "clsx": "^1.2.1",
+    "i18next": "^20.4.0",
+    "react": "^18.2.0",
+    "react-i18next": "^11.18.1",
+    "react-router-dom": "^6.3.0",
+    "tailwindcss": "^3.3.3"
+  },
+  "devDependencies": {
+    "@ovh-ux/manager-vite-config": "*",
+    "@playwright/test": "^1.34.3",
+    "@types/jest": "^29.5.5",
+    "ts-jest": "^29.1.1",
+    "typescript": "^4.3.2",
+    "vite": "^3.0.2"
```

Comment:
Latest version is 4.5.0, is it planned to bump ?

Reply:
Yes we planned to bump it 👍